web/docusaurus.config.js

```diff
@@ -93,7 +93,7 @@ module.exports = {
             },
             {
               label: 'Contact',
-              href: 'docs/contact'
+              href: '/docs/contact'
```

Comment:
Hm, how come all others (Getting started, Todo app tutorial, etc) are defined without the preceding slash? Would a more appropriate fix be changing `href` to `to`?

Reply:
oh, I thought trailing slash would be the fix as it seemed to work on my machine but after looking at the docs once more, it seems more appropriate for `to` to be used instead of `href`, my bad
![image](https://user-images.githubusercontent.com/84540554/198628936-4ad7450f-ec69-438d-909a-bb5fb705f3ae.png)

Reply:
https://github.com/wasp-lang/wasp/pull/791 should have the fix as again, i ended up deleting the wrong fork, lol
I'm quite new to contributing on github. Hence, the mess-ups